logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -41,9 +41,12 @@
 
     protected volatile long unreadCount;
 
+    protected volatile long currentSize;
+
     private final CheckpointIO checkpointIO;
     private final PageIOFactory pageIOFactory;
     private final int pageCapacity;
```

Comment:
I see that `currentSize` is a `long` while `maxSize` is an `int` ... any reason?